budget_manager/views.py

```diff
@@ -40,12 +65,17 @@ def get_context_data(self, **kwargs):
         monthly_expense = Expense.objects.filter(user=self.request.user).annotate(
             month=ExtractMonth('expense_date')).values('month').annotate(total_amount=Sum('cost'))
 
+
+
         context = {
             'total_expense': total_expense,
             'total_income': total_income,
             'total_balance': total_balance,
             'monthly_income': monthly_income,
             'monthly_expense': monthly_expense,
+            "currency_filter_form": currency_filter_form,
+            "displayed_currency": Currency.objects.get(pk=1 if currency_filter_form.data.__len__() is 0 \
```

Comment:
declare a variable outside of dictionary

Reply:
I will put it in the trello board and do it later. Probably tomorrow. Now I want to test something. I will do three separate pull request on three branches from the same main branch state. I am curious about merging stuff like this. It will be great if merging one pull request will merge all of them if there is no conflicts between them. We will see